File: Java_Programming/Practice/LB_class/program9.java
/*
    START
        Accept number as No
        If No is completely divisible by 2
            then print Even
        otherwise
            print Odd
    STOP


    START 
        Accept number as No
        Divide No by 2
        If remainder is 0
            then print as Even
        otherwise
            print as Odd
    STOP
*/

package Practice.LB_class;

import java.util.Scanner;

class program9 
{
   public static void main(String A[])
   {
        int iValue = 0;
        int iRemainder = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        iValue = sobj.nextInt();

        iRemainder = iValue % 2;

        if(iRemainder == 0)
        {
            System.out.println("Number is even");
        }
        else
        {
            System.out.println("Number is Odd");
        }
   } 
}

File: C_Programming/Practice/LB_Class/program9.c
/*
    START 
        Accept number as No
        Divide No by 2
        If remainder is 0
            then print as Even
        otherwise
            print as Odd
    STOP
*/

#include<stdio.h>

void CheckEvenOdd(int iNo)
{
    int iRemainder = 0;

    iRemainder = iNo % 2;

    if (iRemainder == 0)
    {
        printf("Number is Even \n");
    }
    else
    {
        printf("Number is Odd \n");
    }

}

int main()
{
    int iValue = 0;

    printf("Enter number : \n");
    scanf("%d",&iValue);
   
    CheckEvenOdd(iValue);

    return 0;
}